instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -220,14 +222,16 @@ def _test_trace_error(self, code, exc):
         spans = self.get_finished_spans()
         self.assertEqual(1, len(spans))
         span = spans[0]
-        self.assertFalse(span.status.is_ok)
-        self.assertEqual(span.status.status_code, code)
-        self.assertEqual(
-            span.status.description, f"{type(exc).__name__}: {exc}"
-        )
+        if major_version < 8:
```

Comment:
this needs to be resolved but I'm clueless, as the FIXME says span status is not set properly with es8

Reply:
It looks like now the exception raised by the transport is catched somewhere and before it escaped up to the tracer where the span status was updated accordingly.

Reply:
I think I understood this at last! At least for elasticsearch 8 we need to handle failure manually in the instrumentation because we are mocking at the transport layer but exceptions are raised at the elasticsearch layer so after we created our span. Need to check if it's working fine for older versions :sweat_smile: